### ISSUE TITLE
Docs incomplete for sidebar settings?

### ISSUE THREAD

**Issue by montecruiseto** [Semantic-Org/Semantic-UI#1528]
In the [docs for the sidebar settings] (http://semantic-ui.com/modules/sidebar.html#/settings) there seem to be some elements missing, am I right @jlukic ?

It took me a while figuring out through trial and error how to make my sidebar "pushable". Eventually I found that this worked for me:
```
.sidebar('setting', {
transition: 'push'
})
```

In fact, from the semantic source code it seems that the following attributes could be added to the Sidebar settings table in the docs:

```
  transition 
  mobileTransition
  closable    
  dimPage   
  scrollLock
  returnScroll 
  easing 
```

Also, the docs could surely use some instructions on how to invoke these settings, as to avoid having to go through trial-and-error to make it work properly.

**Comment by jlukic** [comment]
Fixed in ``1.5.1``

**Comment by Lalrinfela** [comment]
The scrollLock cannot work. Anybody experience this too?
